rllib/offline/dataset_reader.py

```diff
@@ -25,48 +27,79 @@ def _get_resource_bundles(config: AlgorithmConfigDict):
     return [{"CPU": math.ceil(parallelism * cpus_per_task)}]
 
 
+def _unzip_if_needed(paths: List[str], format: str):
+    """If a path in paths is a zip file, unzip it and use path of the unzipped file"""
+    ret = []
+    for path in paths:
+        fpath = Path(path).absolute()
+        if not fpath.exists():
+            fpath = Path(__file__).parent.parent / path
+        if not fpath.exists():
+            raise FileNotFoundError
```

Comment:
Sorry I didn't write the entire code. Can you give more descriptive error message in FileNotFoundError(msg)?